cmd/image-builder/README.md

```diff
@@ -189,11 +188,11 @@ Image Builder supports three build backends:
 - ADO pipelines
 
 kaniko and BuildKit build images locally, while for the ADO pipelines backend, Image Builder calls ADO API to start the build process.
-To use the kaniko backend, use the `image-builder` image.
-To use the BuildKit backend, use the `buildkit-image-builder` image.
-The ADO backend is supported by both images. To use it, you need to provide the `--build-in-ado=true` flag.
+To use the kaniko backend, use the `image-builder` image and set the `build-in-ado` flag to `false`.
+To use the BuildKit backend, use the `buildkit-image-builder` image and set `build-in-ado` falg to `false`.
+The ADO backend is supported by both images. This is the default backend.
+The preferred and default way to build images is to use the ADO backend because it's the only SLC-29 compliant backend.
```

Comment:
```suggestion
To use the kaniko backend, use the `image-builder` image and set the **build-in-ado** flag to `false`.
To use the BuildKit backend, use the `buildkit-image-builder` image and set the **build-in-ado** flag to `false`.
The ADO backend is supported by both images. 
The preferred and default way to build images is to use the ADO backend because it's the only SLC-29 compliant backend.
```

Reply:
I removed `This is the default backend.` from line 193 as it's basically repeated in line 194. Please bring it back if you disagree.

Reply:
By the way, please use bold font for flags, e.g., the **build-in-ado** flag, and code font for values and pairs of flags with values, e.g., the `--ado-preview-run=true` flag.